windows_docker_resources/rti_web_binaries_install.ps1

```diff
@@ -0,0 +1,20 @@
+ write-output ROS_DISTRO $Env:ROS_DISTRO
+ write-output CONNEXTDDS_DIR $Env:CONNEXTDDS_DIR
+ if ( $Env:ROS_DISTRO -eq "jammy" -or $Env:ROS_DISTRO -eq "humble" ) {
+	pixi run 7z x -oC:\connext "C:\TEMP\rticonnextdds-src\openssl-1.1.1k-target-x64Win64VS2017.zip"
+	& "C:\TEMP\rticonnextdds-src\rti_connext_dds-6.0.1-pro-host-x64Win64.exe" @("--mode", "unattended", "--unattendedmodeui", "minimalWithDialogs". "--prefix", "C:\connext")
+	cmd.exe /c $Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat -u "C:\TEMP\rticonnextdds-src\openssl-1.1.1k-6.0.1.25-host-x64Win64.rtipkg"
+	cmd.exe /c $Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat -u "C:\TEMP\rticonnextdds-src\rti_connext_dds-6.0.1-pro-target-x64Win64VS2017.rtipkg"
+	cmd.exe /c $Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat -u "C:\TEMP\rticonnextdds-src\rti_security_plugins-6.0.1.25-host-x64Win64.rtipkg"
+	cmd.exe /c $Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat -u "C:\TEMP\rticonnextdds-src\rti_security_plugins-6.0.1.25-target-x64Win64VS2017.rtipkg"
+} else {
+	$params = @("--mode", "unattended", "--unattendedmodeui", "minimalWithDialogs", "--prefix", "C:\connext")
+	& "C:\TEMP\rticonnextdds-src\rti_connext_dds-7.3.0-pro-host-x64Win64.exe" $params
+	$rti_security_target_params = @()
+
+	cmd.exe /c "$Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat" "-u" "C:\TEMP\rticonnextdds-src\openssl-3.0.12-7.3.0-host-x64Win64.rtipkg"
+	cmd.exe /c "$Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat" "-u" "C:\TEMP\rticonnextdds-src\openssl-3.0.12-7.3.0-target-x64Win64VS2017.rtipkg"
+	cmd.exe /c "$Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat" "-u"  "C:\TEMP\rticonnextdds-src\rti_connext_dds-7.3.0-pro-target-x64Win64VS2017.rtipkg"
+	cmd.exe /c "$Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat" "-u"  "C:\TEMP\rticonnextdds-src\rti_security_plugins-7.3.0-host-x64Win64.rtipkg"
+	cmd.exe /c "$Env:CONNEXTDDS_DIR\bin\rtipkginstall.bat" "-u" "C:\TEMP\rticonnextdds-src\rti_security_plugins-7.3.0-target-x64Win64VS2017.rtipkg"
```

Comment:
@lobolanja I could use a second set of eyes here. I'm getting 
```
The system cannot find the path specified.
```

Errors trying to run these commands but as far as I can see, they're the same invocation as was in the original Dockerfile.